golem-cli/src/oss/factory.rs

```diff
@@ -74,6 +80,9 @@ impl OssServiceFactory {
             worker_url,
             service_http_client_config: HttpClientConfig::new_for_service_calls(allow_insecure),
             health_check_http_client_config: HttpClientConfig::new_for_health_check(allow_insecure),
+            file_download_http_client_config: HttpClientConfig::new_for_file_download(
+                allow_insecure,
+            ),
```

Comment:
May be, it's a good idea to rename `new_for_file_download`?. It doesn't really expose the purpose of that function.

Reply:
Can you provide a suggestion? This is used to build a client config specific to fetching remote files, so I'm struggling a bit to come up with a better name.